challenges/objects-arrays.js

```diff
@@ -59,11 +97,21 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+function getInfo(graduates){
+  contactInfo.push(`${graduates.first_name} ${graduates.email}`)
+  return contactInfo;
+};
+
+graduates.forEach(getInfo);
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = graduates.filter((graduates) => {
```

Comment:
Keep practicing with arrow syntax, this looks great. Make sure to also change the name of the iterator parameter to singular (graduates -> graduate) to be more semantic. If you want to simplify further and since there is only one return statement you can do this:

```
const uni = graduates.filter(graduate => graduate.university.includes('Uni'));
```